### PR TITLE
Completely new

### PR DESCRIPTION
**Pull request by sschzrk**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -2,21 +2,111 @@
...
+  console.log();
+  console.log('Usage example:\n\r 5 5 + 9 *');
+}
+
+if(operator == 0){
+  console.log("No valid operator!");
+  console.log('Supported operators:\n\r +, -, *, /');
+  console.log();
+  console.log('Usage example:\n\r 5 5 + 9 *');
+}
```
Comment (gh03ty) [inline index.js]: Nice Idea to implement this feature to improve userfriendliness. Also fixed issue #5.

File: index.js [hunk 2]
```diff
@@ -2,21 +2,111 @@
...
 
-var result;
+if(operand.length == operator.length){
+  console.log('Too many operators passed!');
+}
+
+if(operand.length > operator.length+1){
+  console.log('Too less operators passed!');
+}
```
Comment (gh03ty) [inline index.js]: Added failsave for malformed arguments entered while usage of [index.js ](https://github.com/dwdab/ws20-devops-calc/blob/main/index.js)

File: index.js [hunk 3]
```diff
@@ -2,21 +2,111 @@
...
 
-var operand1 = arguments[0];
-var operand2 = arguments[1];
-var operator = arguments[2];
+let operator = [];
+let operand = [];
 
-console.log('Calculating:', operand1, operand2, operator);
+for (let k = 2; k < arguments.length; k++) {
+  if (k % 2 == 0) {
```
Comment (dwdab) [inline index.js]:
```suggestion
  if (k % 2 === 0) {
```

File: index.js [hunk 4]
```diff
@@ -2,21 +2,111 @@
...
+    operand.push(arguments[i]);
+  }
+
+  if(i==0){
+    operand.push(arguments[i]);
+  }
+}
+
+if(arguments == 0){
+  console.log('Supported operators:\n\r +, -, *, /');
```
Comment (dwdab) [inline index.js]: `\n` is usually sufficient.

```suggestion
  console.log('Supported operators:\n +, -, *, /');
```

File: index.js [hunk 5]
```diff
@@ -2,21 +2,111 @@
...
+  console.log();
+  console.log('Usage example:\n\r 5 5 + 9 *');
+}
+
+if(operator == 0){
```
Comment (dwdab) [inline index.js]:
```suggestion
if(operator === 0){
```

**Pull request closed without merging**